main.go

```diff
@@ -97,43 +97,49 @@ func main() {
 }
 
 func federate(ctx context.Context, w http.ResponseWriter, r *http.Request, apiClient v1.API) {
-	// TODO: extend to support multiple match queries
-	// This will only act based on first match query
 	params := r.URL.Query()
-	matchQuery := params.Get("match[]")
+	matchQueries := params["match[]"]
 
+	// TODO: shuld the timeout increase with the query count?
+	// Should it be per query?
+	// Should we make it configurable?
```

Comment:
I think we can make it configurable, please add a todo for it inside the README.

Reply:
Sure, will do.

Reply:
Done, do you want me to remove it from here since it's in the README now?

Reply:
Yes please, as the todo explains the problem, we can remove it. Thank you